src/android/app/src/main/java/io/github/lime3ds/android/display/ScreenAdjustmentUtil.kt

```diff
@@ -56,4 +61,17 @@ class ScreenAdjustmentUtil(
         NativeLibrary.reloadSettings()
         NativeLibrary.updateFramebuffer(NativeLibrary.isPortraitMode)
     }
+
+    fun changeActivityOrientation(orientationOption: Int) {
+        if (currentOrientation == orientationOption) {
+            return
+        }
```

Comment:
Why is this necessary? What happens if you set the orientation to the current one which we are trying to avoid?